airflow/models/dagbag.py

```diff
@@ -492,13 +492,14 @@ def collect_dags(
         Given a file path or a folder, this method looks for python modules,
         imports them and adds them to the dagbag collection.
 
-        Note that if a ``.airflowignore`` file is found while processing
+        Note that if an ``.airflowignore`` file is found while processing
```

Comment:
For me _a_ is right because `.airflowignore` is prodounced _dot airflow ignore_.

Reply:
Hmm, yeah, I can go long with that :) Will update.